### ISSUE TITLE
Celery in local development appears frozen

### ISSUE THREAD

**Issue by chouseknecht** [ansible/galaxy#1402]
Working in local dev environment. Built from scratch using latest code from `devel` branch. After connecting to GitHub, attempted to import a test repository via My Content page. 

Celery is not logging any heartbeat, nor is it logging anything about the requested imports. Here's a screen shot of the console:

![celery_logs](https://user-images.githubusercontent.com/1282865/49616248-31b5e580-f97e-11e8-8576-15d72d684a2e.png)

Nothing displays under My Imports:

![stuck_import_01](https://user-images.githubusercontent.com/1282865/49616276-472b0f80-f97e-11e8-8354-2f4153d6c01c.png)

And nothing updating on My Content:

![pending_02](https://user-images.githubusercontent.com/1282865/49616299-590cb280-f97e-11e8-8ce7-86b55ab09438.png)

**Comment by ironfroggy** [comment]
I can confirm the same behavior observed for my builds. However, I checked out branch `stable/3.1` for comparison and _did_ successfully import. @alikins and I also both witnessed a specific `bytes` vs `str` error when forcing a task to run, indicating a Python 2/3 issue. Since this was the most likely key difference between `stable/3.1` and `devel` I've tried a more narrow comparison test: the `devel` branch with the Python 3 commit reverted. Now I can confirm the celery behavior is definitely caused by the Python version change, likely stemming from some dependency versions that are incompatible with this the target of Python 3.6.6.
